database/seeds/BodySeeder.php

```diff
@@ -17,67 +17,368 @@ class BodySeeder extends Seeder
      */
     public function run()
     {
-        Address::create([
-            'country_id'    =>  21,
-            'street'        =>  'Notelaarsstraat 55',
-            'zipcode'       =>  '1000',
-            'city'          =>  'Brussels',
-        ]);
-        Address::create([
-            'country_id'    =>  153,
-            'street'        =>  'Oude Markt 24',
-            'zipcode'       =>  '7511 GB',
-            'city'          =>  'Enschede',
-        ]);
-        Address::create([
-            'country_id'    =>  81,
-            'street'        =>  'Bergstraße 66c',
-            'zipcode'       =>  '01069',
-            'city'          =>  'Dresden',
+	    BodyType::create([
+            'name'          =>  'antenna', //1
         ]);
 
         BodyType::create([
-            'name'          =>  'local',
+            'name'          =>  'contact antenna', //2
         ]);
 
         BodyType::create([
-            'name'          =>  'special',
+            'name'          =>  'contact', //3
+        ]);
+
+	    BodyType::create([
+	        'name'	    =>  'commission', //4
+	    ]);
+
+	    BodyType::create([
+	        'name'	    =>  'committee', //5
         ]);
 
         BodyType::create([
-            'name'          =>  'commission-european',
+            'name'      =>  'special', //6
+        ]);
+	
+
+	    Address::create([ //TODO: Validate address
+            'country_id'    =>  202,
+            'street'        =>  'Campus de Elviña',
+            'zipcode'       =>  '15071',
+            'city'          =>  'A Coruna',
         ]);
 
     	Body::create([
-            'type_id'       =>  2,
+            'type_id'       =>  1,
             'address_id'    =>  1,
-    		'name'			=>	'AEGEE-Europe',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'AEU' //No clue if this is actually correct.
+    	    'name'	        =>	'AEGEE-A Coruna',
+            'email'         =>  'aegee.coruna@gmail.com',
+	        //'phone'	    =>  '',
+            'legacy_key'    =>  'TMP' //TODO
     	]);
 
+	
+        Address::create([
+            'country_id'    =>  81,
+            'street'        =>  'Templergraben 55',
+            'zipcode'       =>  '52056',
+            'city'          =>  'Aachen',
+        ]);
+
     	Body::create([
             'type_id'       =>  1,
             'address_id'    =>  2,
-    		'name'			=>	'AEGEE-Enschede',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'ENS' //No clue if this is actually correct.
+    	    'name'	        =>	'AEGEE-Aachen',
+            'email'         =>  'info@aegee-aachen.org',
+	        'phone'	        =>  '2418097121',
+            'legacy_key'    =>  'TMP' //TODO
     	]);
 
+
+	    Address::create([ //TODO: Address unknown
+            'country_id'    =>  220,
+            'street'      =>  '',
```

Comment:
For now this works, yet I am unsure whether instead we should make street / zipcode optional. On one hand this would fix the address issue, on the other hand it does not really make sense to not know the local's address.
However, for people it might be useful to make it optional: If you do not want to share your street/zipcode/city.

So end conclusion on my opinion: I think it makes sense to have all of the fields (even country_id) optional.